dev-python/backports-shutil_which/backports-shutil_which-3.5.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Sorry, this was actually copied from another ebuild, so I did not pay attention to that. Thanks for the tip.